CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - BREAKING CHANGE: `tx.send()` is now asynchronous: old: `send(): TransactionId` new: `send(): Promise<TransactionId>` and `tx.send()` now directly waits for the network response, as opposed to `tx.send().wait()`
 - `Circuit.witness` can now be called outside circuits, where it will just directly return the callback result
 - BREAKING CHANGE: Static methods of type `.fromString()`, `.fromNumber()` and `.fromBigInt()` on `Field`, `UInt64`, `UInt32` and `Int64` are not longer supported.
+- Rename `SmartContract.experimental.authorize()` to `SmartContract.experimental.approve()`
```

Comment:
`authorize` wasn't released yet - it's in the "Added" section! could you remove this and change the name in the "Added" section?

Reply:
Will do!